src/Rules/Gedmo/PropertiesExtension.php

```diff
@@ -0,0 +1,88 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Gedmo;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Gedmo\Mapping\Annotation as Gedmo;
+use PHPStan\Reflection\PropertyReflection;
+use PHPStan\Rules\Properties\ReadWritePropertiesExtension;
+use function get_class;
+use function in_array;
+
+class PropertiesExtension implements ReadWritePropertiesExtension
+{
+
+	private const GEDMO_WRITE_CLASSLIST = [
+		Gedmo\Blameable::class,
+		Gedmo\IpTraceable::class,
+		Gedmo\Locale::class,
+		Gedmo\Language::class,
+		Gedmo\Slug::class,
+		Gedmo\SortablePosition::class,
+		Gedmo\Timestampable::class,
+		Gedmo\TreeLeft::class,
+		Gedmo\TreeLevel::class,
+		Gedmo\TreeParent::class,
+		Gedmo\TreePath::class,
+		Gedmo\TreePathHash::class,
+		Gedmo\TreeRight::class,
+		Gedmo\TreeRoot::class,
+		Gedmo\UploadableFileMimeType::class,
+		Gedmo\UploadableFileName::class,
+		Gedmo\UploadableFilePath::class,
+		Gedmo\UploadableFileSize::class,
+	];
+
+	private const GEDMO_READ_CLASSLIST = [
+		Gedmo\Locale::class,
+		Gedmo\Language::class,
+	];
+
+	/** @var AnnotationReader */
+	private $annotationReader;
+
+	public function __construct()
+	{
+		$this->annotationReader = new AnnotationReader();
```

Comment:
Maybe we could check if this class exists, and if it doesn't, all of the extension methods would return false.

Reply:
Fixed!